jwql/instrument_monitors/nirspec_monitors/ta_monitors/msata_monitor.py

```diff
@@ -57,8 +58,7 @@
 from jwql.utils import monitor_utils
 from jwql.utils.constants import JWST_INSTRUMENT_NAMES_MIXEDCASE
 from jwql.database.database_interface import session, engine
-from jwql.database.database_interface import NIRSpecTAQueryHistory, NIRSpecTAStats
-from jwql.utils.utils import ensure_dir_exists, filesystem_path, get_config, filename_parser
+from jwql.utils.utils import ensure_dir_exists, filesystem_path, get_config
 
 
 class MSATA():
```

Comment:
This file has some conflicts with changes on develop. I can help resolve them if you like.